database/migrations/2016_11_03_220633_create_general_orderings_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateGeneralOrderingsTable extends Migration {
+
+	public function up()
+	{
+		Schema::create('general_orderings', function(Blueprint $table) {
+			$table->increments('id');
+			$table->timestamps();
+			$table->string('name');
+			$table->string('qualified_name');
+			$table->enum('visibility', ['public', 'protected', 'private', 'package']);
+			$table->enum('specifications_id', array('message_occurrence_specifications_id', 'execution_occurrence_specifications_id'));
+			$table->enum('specifications_type', array('message_occurrence_specifications', 'execution_occurrence_specification'));
```

Comment:
to ja chápem ale ide mi o to, že specifications_id môže mať 2 hodnoty, ale čo to znamená keď tam bude napríklad 'message_occurrence_specifications_id'? ako zistím, ku akému ID sa viaže daný riadok ?
Polymorpické vzťahy sú také kde si uložím meno konkrétneho modelu resp. jeho alias (specifications_type), čo je dobre, ale navyše potrebujem vedieť ID (integer), na ktorý riadok sa viaže https://laravel.com/docs/5.3/eloquent-relationships#polymorphic-relations